js_modules/dagit/packages/core/src/nav/RepositoryLocationStateObserver.tsx

```diff
@@ -78,8 +78,7 @@ export const RepositoryLocationStateObserver = () => {
               underline="always"
               onClick={() => {
                 setUpdatedLocations([]);
-                refetch();
-                client.resetStore();
+                client.refetchQueries({include: 'active'});
```

Comment:
why do we need `refetchQueries` here? Can we narrow down the queries being fetched somehow? My concern is that  we might refetch some expensive query added later that doesn't need to be refetched